pages/common/truffle.md

```diff
@@ -0,0 +1,24 @@
+# truffle
+
+> A tool for developing smart contracts.
```

Comment:
I can't really think of what a `smart contract` should be. Would it be possible to elaborate this a little more?

Reply:
It's an Ethereum thing as far as I know.
Some kind of way to run services on the blockchain. (I think, not exactly an expert.)

https://ethereum.org/en/developers/docs/smart-contracts/

Reply:
I think writing it like this would help a lot:

```suggestion
> A tool for developing smart contracts for running services on the Ethereum blockchain.
```